internal/pkg/diagnostics/diagnostics.go

```diff
@@ -579,7 +580,6 @@ func saveLogs(name string, logPath string, zw *zip.Writer) error {
 func RedactSecretPaths(mapStr map[string]any, errOut io.Writer) map[string]any {
 	v, ok := mapStr["secret_paths"]
 	if !ok {
-		fmt.Fprintln(errOut, "No output redaction: secret_paths attribute not found.")
```

Comment:
When a diagnostics action is ran secrets in the component related files are not redacted (structure does not match, and `secret_paths` is not passed)
This line causes extra noise in the output